.github/actions/reopenIssueWithComment/reopenIssueWithComment.js

```diff
@@ -0,0 +1,31 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+
+const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+const issueNumber = core.getInput('ISSUE_NUMBER', {required: true});
+
+function reopenIssueWithComment() {
```

Comment:
More logs is always a good idea (at least in this context), I promise you won't regret it!